src/data/queries/pressReleaseListing.ts

```diff
@@ -85,25 +87,32 @@ export const data: QueryData<
     totalItems,
     totalPages: totalPages,
     current: opts?.page,
+    lovell: opts.context?.lovell,
   }
 }
 
 export const formatter: QueryFormatter<
   PressReleaseListingData,
   PressReleaseListing
-> = ({ entity, releases, menu, totalItems, totalPages, current }) => {
+> = ({ entity, releases, menu, totalItems, totalPages, current, lovell }) => {
   const formattedReleases = releases.map((release) => {
     return queries.formatData(
       `${RESOURCE_TYPES.PRESS_RELEASE}--teaser`,
       release
     )
   })
 
-  const formattedMenu = buildSideNavDataFromMenu(entity.path.alias, menu)
+  let formattedMenu =
+    menu !== null ? buildSideNavDataFromMenu(entity.path.alias, menu) : null
+
+  if (lovell?.isLovellVariantPage) {
+    formattedMenu = getLovellVariantOfMenu(formattedMenu, lovell?.variant)
+  }
+
   return {
     ...entityBaseFields(entity),
     introText: entity.field_intro_text,
-    releases: formattedReleases,
+    'news-releases': formattedReleases,
```

Comment:
Without this, we don't have any releases listed. 🫠 

It's the most annoying thing, but the short version is this: There's a bit in the Lovell-specific logic that expects the URL segment to match this property name. Changing the URL segment constant breaks some tests and doesn't actually render any of the press releases.

The alternative to renaming this is to dig into the Lovell logic and clean it up, but I'm not confident I know what's going on in there to make it so right now. It would also be a lot more work. 😬